### ISSUE TITLE
MMM-GoogleAssistant Update Question

### ISSUE THREAD

**Issue by jmwyds** [bugsounet/MMM-GoogleAssistant#141]
Hey Cedric,

So I am updating the MMM-GoogleAssistant module in accordance with the instructions in the Wiki:
`npm run update`
However, when I run the command, I am getting npm errors and it causes my mirror to crash and open in its [Electron] state. I don't know whether this is a problem with the update script or if this is a problem on my end. However, I have included a couple of pictures to demonstrate the issue more clearly. The first is the terminal output when running npm run update and the second is the log that it directs me to in the error statement.

Thanks for your time!
Jack

Note: I also should also mention that after I was met with this issue, I tried to do a clean clone of the repo and when I got the the 'npm install' instruction of the installation, I was met with problems in the libraries again. However, the issue that the terminal reported was slightly different.

![Screen Shot 2020-09-03 at 9 22 15 AM](https://user-images.githubusercontent.com/26585685/92127840-52b3a080-edc7-11ea-9e65-b8570cdb7863.png)
![Screen Shot 2020-09-03 at 9 27 15 AM](https://user-images.githubusercontent.com/26585685/92128147-b6d66480-edc7-11ea-9bb5-6d63a7d67a83.png)

**Comment by bugsounet** [comment]
Hi, I don't know why you have this error.

Can you try `npm run rebuild` in Google Assistant directory ?
thank for the feellback

**Comment by jmwyds** [comment]
Hey Cedric. Hm, strange. When I ran `npm run rebuild` I received the npm error which I saw after a fresh clone and npm install. I have included a picture of the error and its corresponding log output below:
![Screen Shot 2020-09-03 at 1 44 57 PM](https://user-images.githubusercontent.com/26585685/92154785-f7e07000-edeb-11ea-8b9e-afb23e5f7c16.png)
![Screen Shot 2020-09-03 at 1 48 10 PM](https://user-images.githubusercontent.com/26585685/92154884-22322d80-edec-11ea-91e1-d32e0835eb1b.png)
